src/WordsMove.java

```diff
@@ -59,7 +59,13 @@ public void doExecute(WordsObject object, WordsEnvironment environment) {
 	@Override
 	public LinkedList<WordsAction> doExpand(WordsObject object) {
 		if (distanceExpression != null) {
-			AST.ASTValue value = distanceExpression.eval(null);
+			AST.ASTValue value;
+			try {
+				value = distanceExpression.eval(null);
+			} catch (WordsException e) {
+				System.err.println(e.toString());
+				return null;
```

Comment:
Will returning null here be a problem for actionQueue.addAll() in WordsObject.java?  It seems like we could get stuck in an infinite loop there.  I think the solution is in WordsObject.java to check if it's null, and if it is, return from executeNextAction entirely without attempting to do anything else.  You could pretty easily put that in or open a separate issue about it if you like.